### ISSUE TITLE
Starting with Symfony the best PHP Framework

### ISSUE THREAD

**Issue by fernandofreamunde** [fernandofreamunde/challenge-github-blog#6]
Symfony is a popular PHP framework used for developing web applications. If you're new to Symfony, here's a simple guide to help you get started:

Install Symfony: The first step to using Symfony is to install it on your computer. You can download the installation file from the official Symfony website. Follow the instructions provided to complete the installation process.

Learn the basics: Once you have installed Symfony, it's time to learn the basics. Symfony uses the Model-View-Controller (MVC) pattern, so you should start by learning about controllers, views, and models. You should also learn about Symfony's routing system, which is used to map URLs to controller actions.

Create a new project: Symfony provides a command-line interface (CLI) tool called "Symfony Console" that you can use to create a new project. Open a command prompt and run the following command:

```bash
symfony new myproject --full
```

This command will create a new Symfony project named "myproject" with all the necessary components and dependencies.

Create a new controller: In Symfony, controllers are responsible for handling user requests and returning responses. To create a new controller, open the "src/Controller" directory in your project and create a new PHP file with the following code:

```php
<?php

namespace App\Controller;

use Symfony\Component\HttpFoundation\Response;

class HelloController
{
    public function index()
    {
        return new Response('<html><body>Hello, World!</body></html>');
    }
}
```

This controller returns a simple HTML response with the text "Hello, World!".

Create a new route: Symfony uses a routing system to map URLs to controller actions. To create a new route for our "HelloController", open the "config/routes.yaml" file in your project and add the following code:

```php
hello:
    path: /hello
    controller: App\Controller\HelloController::index
```

This route maps the "/hello" URL to the "index" action of the "HelloController".

Run the application: To run the application, open a command prompt and navigate to the root directory of your project. Run the following command:

```bash
symfony server:start
```

This command will start the Symfony development server. Open a web browser and navigate to "http://localhost:8000/hello". The browser should display the text "Hello, World!".

In conclusion, Symfony is a powerful PHP framework that can help you develop web applications quickly and efficiently. By following the steps outlined above, you can get started with Symfony and start building your own web applications.
